ClosedXML/Excel/XLWorkbook_Load.cs

```diff
@@ -1361,12 +1361,9 @@ private static void ParseReference(string item, out string sheetName, out string
                 else
                     formula = cell.CellFormula.Text;
 
-                if (cell.CellFormula.Reference != null)
-                {
-                    // Parent cell of shared formulas
-                    // Child cells will use this shared index to set its R1C1 style formula
-                    xlCell.FormulaReference = ws.Range(cell.CellFormula.Reference.Value).RangeAddress;
-                }
+                // Parent cell of shared formulas
```

Comment:
Why did you remove this `if`?

Reply:
Here we are already inside the condition 
`if (cell.CellFormula != null && cell.CellFormula.SharedIndex != null && cell.CellFormula.Reference != null)`

Reply:
It would benefit me to expand the preceding code blocks! :-)